generator/app/template/package.json

```diff
@@ -3,25 +3,31 @@
   "version": "0.0.1",
   "description": "app1 microservice",
   "scripts": {
-    "start": "node ./dist/app.js",
-    "test": "mocha ./dist/test",
+    "build": "babel src -d dist --source-maps",
```

Comment:
Mismatch with the direct dependencies. We could
- leave it as is since we add it globally in the image
- change to `atomiq make babel`
- add babel-cli to devDependencies

I'm pretty sure adding babel-cli would be light since babel-core is already a dependency (from babel-register) and it's already in the image cache. I'll take a look into how much weight it adds